frontend/src/components/Header/Header.tsx

```diff
@@ -33,6 +33,7 @@ export const Header: React.FC = () => {
       <Container
         state={headerState}
         className="h-11 bg-primary1 flex flex-row items-center sticky z-header px-3 shadow-sm text-primary3 flex-shrink-0"
+        style={{zIndex:2}}
```

Comment:
We try to avoid `style` props in components (I know there are some in the basecode but we try to improve it). 
The problem with the `style` property is the generation of a new object (without reference) at each rendering, so we generate the style using TailwindCSS and Styled-component.

This component already has a `z-index` defined with `z-header` className (defined here https://github.com/GeotrekCE/Geotrek-rando-v3/blob/main/frontend/tailwind.config.js#L82).

You can replace `z-header` with one of the tailwindCSS api https://v2.tailwindcss.com/docs/z-index or by replacing `z-header` with a new one you create in the tailwind configuration.

Just a warning: have you checked that everything is fine from 500 z-index to 2?